CHANGELOG.md

```diff
@@ -15,7 +18,6 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [3.143.1] - 2021-04-28
 ### Fixed
 - Makes main SearchBar placeholder translatable
```

Comment:
Please revert this change

Reply:
I'm not sure what do you mean. I've updated the changelog. Hope that's it :)

Reply:
WIth the rebase things moved around, but you removed an empty line right below this one. Please put it back :)

Reply:
Oh...yes, my bad. Thank you!